engines/dreamweb/stubs.cpp

```diff
@@ -1273,13 +1292,23 @@ void DreamWebEngine::commandWithOb(uint8 command, uint8 type, uint8 index) {
 
 	if (getLanguage() != Common::RU_RUS) {
 		printDirect(string, _textAddressX, _textAddressY, textLen, (bool)(textLen & 1));
+    	if (ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
+			const char *temp = (const char*)string;
```

Comment:
What is the purpose of this variable, why not add `string` right away to `theText`?

Reply:
Hello sev, thank you for reviewing. For your question: Because it would make the uttered speech sound more natural. For example, "Walk to Jeans", instead of saying: "Walk to", "Jeans", it would say: "Walk-tuh-jeans", so I decided to concatenate before saying.

Reply:
The question is not about the `theText` variable that you are using to concatenate the verb and the object, but about the `temp` variable. You could write directly `theText += (const char *)string` without using a temporary variable.

And by the way you are missing a space before the `*` in `(const char*)string`.